docs/how-to/unified_memory.rst

```diff
@@ -0,0 +1,595 @@
+.. meta::
+  :description: This chapter describes introduces Unified Memory (UM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, unified memory, unified, memory, UM, APU
+
+*******************************************************************************
+Unified Memory
+*******************************************************************************
+
+Concept
+=======
+
+In a conventional architectures, both CPUs a GPUs have a dedicated memory,
```

Comment:
Thanks, Matyas. Appreciate the prompt fixes. 

Some more minor fixes.

Reply:
Thank you! I applied the recommended fixes.